features/user_sessions.feature

```diff
@@ -66,9 +66,22 @@ Feature: User sessions
       | contact_name | email            | password  | password_updated_at | approval_status |
       | Jane         | jane@example.com | test1234! | 2017-01-01          | approved        |
     And a clear email queue
-    And the date is 2017-01-02
+    And the time is 2017-03-30 23:55
     And I log in with email "jane@example.com" and password "test1234!"
-    And the date becomes 2017-04-15
```

Comment:
Let's use `And the time becomes` rather than `And the time is` in this line. Functionally they are equivalent, but `And the time becomes` is a better indicator of the passage of time that we want to capture in this feature.